src/cdomains/intDomain.ml

```diff
@@ -648,12 +648,24 @@ struct
   | `Definite x -> if i = x then `Eq else `Neq
   | `Excluded (s,r) -> if S.mem i s then `Top else `Neq
 
-  let cast_to t = function
+  let top_of ik = `Excluded (S.empty (), size ik)
+  let top_if_not_in_int64 ik f x = try f x with Size.Not_in_int64 -> top_of ik
+  let cast_to ?torg ik = top_if_not_in_int64 ik @@ function
```

Comment:
I think this `?torg` may be benefit from some comments, it's not completely clear what it means to me

Reply:
I'll put in in the .mli since that's where most comments are.
Aside: It's a bit annoying to have two places for it.
~~~sh
$ fd .mli   
src/analyses/libraryFunctions.mli
src/cdomains/arrayDomain.mli
src/cdomains/arrayDomain_deprecated.mli
src/cdomains/intDomain.mli
src/cdomains/structDomain.mli
~~~
Not that many left...